### PR TITLE
Fix performance for the Safari browser

### PR DESCRIPTION
**Pull request by smyzin**
- moved `drop-shadow` from svg to the wrapper
- added `isolate` property
- added `will-change` property
- added additional classes to the Arc Text to prevent jumping

### CODE REVIEW

File: src/components/DosuGraphicsBlock.tsx [hunk 1]
```diff
@@ -1,12 +1,27 @@
...
-const arcTextWrapper = classnames(
-  position('absolute'),
-  inset('right-3', '-bottom-11', 'md:left-14', 'md:right-auto')
-)
+const arcTextWrapper = (visible: boolean) =>
+  classnames(
+    position('absolute'),
+    inset('right-3', '-bottom-11', 'md:left-14', 'md:right-auto'),
+    opacity({ 'opacity-0': !visible }),
+    willChange('will-change-auto'),
```
Comment (T-Damer) [inline src/components/DosuGraphicsBlock.tsx:21]: Will change opacity is better here

File: src/components/DosuGraphicsBlock.tsx [hunk 2]
```diff
@@ -17,9 +32,11 @@ const wrapper = classnames(
 )
 
 export default function () {
+  const { ref, inView } = useInView()
   return (
```
Comment (T-Damer) [inline src/components/DosuGraphicsBlock.tsx:36]: Let's add a blank line here

File: src/components/Footer.tsx [hunk 3]
```diff
@@ -1,11 +1,11 @@
-import { display } from 'classnames/tailwind'
+import { opacity } from 'classnames/tailwind'
 import { useInView } from 'react-intersection-observer'
 import classNamesToString from 'helpers/classNamesToString'
 
 export default function () {
   const { ref, inView } = useInView()
 
-  const grid = classNamesToString(display({ hidden: !inView }), 'grid')
+  const grid = classNamesToString(opacity({ 'opacity-0': !inView }), 'grid')
```
Comment (T-Damer) [inline src/components/Footer.tsx:8]: I guess we also need `visibility-hidden` there

File: src/icons/OctocorpLogo.tsx [hunk 4]
```diff
@@ -13,41 +13,42 @@ const blueStroke = classnames(stroke('stroke-primary'), strokeWidth('stroke-4'))
...
-      <path
-        d="M74.1182 108.266C71.7799 104.01 66.379 95.4966 63.4824 95.4966C59.8617 95.4966 58.504 110.731 74.1182 108.266Z"
-        className={blueStroke}
-      />
-      <path
-        d="M79.8867 108.266C82.225 104.01 87.6259 95.4966 90.5225 95.4966C94.1432 95.4966 95.5009 110.731 79.8867 108.266Z"
-        className={blueStroke}
-      />
-    </svg>
+    <div className={octoCorpContainer}>
```
Comment (T-Damer) [inline src/icons/OctocorpLogo.tsx:16]: Does this really work? Interesting 🤔

**Review by T-Damer — APPROVED** [review]

File: src/icons/OctocorpLogo.tsx (hunk 4 above)
Comment (smyzin) [reply src/icons/OctocorpLogo.tsx:16]: Yep
Снимок экрана 2022-11-10 в 12 46 20

File: src/components/Footer.tsx (hunk 3 above)
Comment (smyzin) [reply src/components/Footer.tsx:8]: Added

File: src/components/DosuGraphicsBlock.tsx (hunk 2 above)
Comment (smyzin) [reply src/components/DosuGraphicsBlock.tsx:36]: Added

**Review by T-Damer — APPROVED** [review]

**Pull request merged**